internal/test/framework/framework.go

```diff
@@ -242,6 +243,67 @@ func (f *TestFramework) getClusterVersion() error {
 	return nil
 }
 
+// GetK8sVersion gets the Kubernetes version in vx.x.x format
+func (f *TestFramework) GetK8sVersion() (string, error) {
+	versionInfo, err := f.OSConfigClient.Discovery().ServerVersion()
```

Comment:
You can make versionInfo a field, if you don't want make api calls to OpenShift apiserver

Reply:
Yeah I agree. Add it to the framework struct and populate during the initialization.